src/repositories/dbCronRepository.ts

```diff
@@ -33,6 +40,70 @@ export const schedulePowerBoostingSnapshot = async (
   `);
 };
 
+export const invokeGivPowerHistoricProcedures = async () => {
+  const connection = getConnection();
+  await connection.query(`
+    CALL public."TAKE_POWER_BOOSTING_SNAPSHOT_HISTORY"()
+  `);
+  await connection.query(`
+    CALL public."TAKE_POWER_BALANCE_SNAPSHOT_HISTORY"()
+  `);
+  await connection.query(`
+    CALL public."TAKE_POWER_SNAPSHOT_HISTORY"()
+  `);
+};
+
+export const schedulePowerBoostingSnapshotHistory = async (
+  cronJobExpression: string,
+) => {
+  const connection = getConnection('cron');
+  await connection.query(`
+      CREATE EXTENSION IF NOT EXISTS PG_CRON;
+
+      GRANT USAGE ON SCHEMA CRON TO POSTGRES;
+
+      SELECT CRON.SCHEDULE(
+        '${POWER_BOOSTING_SNAPSHOT_HISTORY_TASK_NAME}',
+        '${cronJobExpression}',
+        $$CALL public."TAKE_POWER_BOOSTING_SNAPSHOT"()$$
+      );
+  `);
+};
+
+export const schedulePowerBalanceSnapshotHistory = async (
+  cronJobExpression: string,
+) => {
+  const connection = getConnection('cron');
+  await connection.query(`
+      CREATE EXTENSION IF NOT EXISTS PG_CRON;
+
+      GRANT USAGE ON SCHEMA CRON TO POSTGRES;
+
+      SELECT CRON.SCHEDULE(
+        '${POWER_BALANCE_SNAPSHOT_HISTORY_TASK_NAME}',
+        '${cronJobExpression}',
+        $$CALL public."TAKE_POWER_BOOSTING_SNAPSHOT"()$$
+      );
+  `);
+};
+
+export const schedulePowerSnapshotHistory = async (
+  cronJobExpression: string,
+) => {
+  const connection = getConnection('cron');
+  await connection.query(`
+      CREATE EXTENSION IF NOT EXISTS PG_CRON;
+
+      GRANT USAGE ON SCHEMA CRON TO POSTGRES;
+
+      SELECT CRON.SCHEDULE(
+        '${POWER_SNAPSHOT_HISTORY_TASK_NAME}',
+        '${cronJobExpression}',
+        $$CALL public."TAKE_POWER_BOOSTING_SNAPSHOT"()$$
+      );
+  `);
+};
+
```

Comment:
As mentioned in another comment, let's have only one procedure and one expression.

Reply:
Done